src/notifications/hook_gamefull.py

```diff
@@ -0,0 +1,48 @@
+from PyQt5 import QtCore
+import util
+import config
+from config import Settings
+from notifications.ns_hook import NsHook
+import notifications as ns
+
+"""
+Settings for notifications: if a player comes online
+"""
+
+
+class NsHookGameFull(NsHook):
+    def __init__(self):
+        NsHook.__init__(self, ns.Notifications.GAME_FULL)
+        self.button.setEnabled(True)
```

Comment:
Since you don't have any settings to modify, I suggest just not doing anything here except calling NsHook.__init__. The 'more' button will remain greyed out this way. All code below can be removed too.